### ISSUE TITLE
No communication with KPro 

### ISSUE THREAD

**Issue by dylanmilesmsu** [pablobuenaposada/HonDash#263]
Raspberry Pi 4 Model B
Hondash 3.5.0 image 
KPro V4, running software 4.5.8.0
Hondash WAP appears, I can connect and view hondash.local
It shows KPro v0.00, all values in browser show 0
I don't have a screen for my Pi (yet), so testing with my phone viewing hondash.local
If I unplug my KPro, it will say "Unknown". So it does correctly detect that it is KPro, but can't figure out the version or any other information from it.
I had the same issue when previously running it on AML-S905X-CC, but I was told "It works, just not on unsupported systems like AML-S905X-CC"
However now I have a supported system, RasPi 4 Model B, same exact problem. 

Also worth mentioning, going to hondash.local/setup just redirects to hondash.local/basic.html, but I think that is a separate issue.

If there are any log files, let me know and I will look for them.

![IMG20240930161225](https://github.com/user-attachments/assets/0bfd3cd2-9f53-42e2-ace2-2d6dfe702f57)

![Screenshot_2024-09-30-16-14-14-76_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/cc5b9174-5c62-4412-b3a3-c57767c263c4)

![IMG20240930161419](https://github.com/user-attachments/assets/c56261b5-5153-4770-8ac2-c313ff035462)

**Comment by dylanmilesmsu** [comment]
Got my screen in, same problem:
https://github.com/user-attachments/assets/c0de183c-6212-4ab5-9a63-894be8c96d1c
I've tried plugging it in before turning the pi on, plugging it in afterwards, using different usb ports, I get nothing.

**Comment by pablobuenaposada** [comment]
Thanks for the detailed explanation, for the setup thing try setup**/** instead of setup
I will create a new image with the latest Raspbian and the latest version of everything used to see if fixes the issue.
I will try to also incorporate logs, give me a few days.

**Comment by Thesomer** [comment]
Hola, estoy exactamente con el mismo problema, se sabe de alguna solucion?.

**Comment by pablobuenaposada** [comment]
ok, here it is a refreshed version, hoping it fixes some of these issues, let me know if it works so I can release it to the public.
https://www.dropbox.com/scl/fi/mv6q5njitqacxez45riul/HonDash_v3.5.0_refresh.img?rlkey=vhw2j2o592zmzmuf5bz2bghsd&st=hi28gg90&dl=0

**Comment by dylanmilesmsu** [comment]
Thank you very much. I am downloading the image and will flash it shortly. I should have results within a couple of hours.

**Comment by dylanmilesmsu** [comment]
Sadly, I have the same problem as before. I once again tried plugging in the ecu before starting the pi, after it was already running, and cycling the key on and off, but I still only get KPro v0.00 and all zeros shown on screen. I did see a previous user say that the problem is with KPro v4.5+, and while I never tested it before to find out if that is true for myself, it might account for the discrepancy in it working for some users and not for others. If you need anything from me in terms of logs or troubleshooting steps let me know. If you have discord I can even set you up with the ability to SSH into my Pi as its connected to my car so you can troubleshoot as much as you want. 


https://github.com/user-attachments/assets/e301278f-711c-4d40-8106-ad78a20a4a41


https://github.com/user-attachments/assets/b6d55649-84c3-48ef-8484-d915ca977f63

**Comment by pablobuenaposada** [comment]
Ok thanks, that was quick.
This is strange, so kpro 4.5.x doesn't say much because that is the version of kmanager which is kinda irrelevant, but:
- can you go inside kmanager and tell me the firmware version of your board?
- I guess that in manager you can click the lightning icon and see all the values In real time right?
- if you can ssh, can you copy paste me the content of /home/pi/Desktop/HonDash/hondash.log

if have discord right, disclaimer8530

**Comment by pablobuenaposada** [comment]
Thanks to @dylanmilesmsu we figured out what was the problem.
Here's is the next version that I plan to release, would we nice if you guys having trouble can confirm that it works now.
https://www.dropbox.com/scl/fi/vju25j2w7usqjg1iaspij/HonDash_v3.6.0.img?rlkey=foz7en3eim0f87qzfgg0vkm19&st=2ogjmvue&dl=0

**Comment by Thesomer** [comment]
Por mi parte ahora mismo no tengo el coche disponible y no puedo probarlo, pero lo antes posible revisare si funciona y lo comunico, muchas gracias

**Comment by Thesomer** [comment]
Hi, the problem has also been solved for me with the latest version, thanks a lot for the update. Now I have a new problem, with this version the boot is too slow (1 minute 30 seconds) and most of the time it is not able to finish booting (I upload a photo), I use the raspberry pi 3b+ supplied in the kit ready, maybe I should buy a newer version with more ram? Which one would be compatible and faster to use with hodash? thanks for your help
![20241212_145155](https://github.com/user-attachments/assets/15c59c94-4cfb-4439-bbad-9d199c47ae34)

**Comment by Thesomer** [comment]
Thanks for your reply, I think I'm going to buy a new 8gb 64 bit raspberry pi 4 b, would this model be compatible with hondash? any other better alternative? I want to see if this fixes the problem of hanging and not finishing loading

**Comment by pablobuenaposada** [comment]
raspberry pi 4 yes, raspberry pi 5 still pending to try
